imports/ui/components/edit/edit-img.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+import { Meteor } from 'meteor/meteor'
+
+const UploadcareInner = React.createClass({
+  propTypes: {
+    content: React.PropTypes.string,
+    update: React.PropTypes.func,
+    save: React.PropTypes.func
+  },
+  getInitialState: function () {
+    return {
+      src: this.props.content
+    }
+  },
+  componentDidMount: function () {
+    this.update()
+  },
+  update: function () {
+    this.props.update(this.refs.img.src)
+  },
+  uploadcare: function (e) {
+    e.preventDefault()
+    let self = this
+    let opts = {
+      crop: 'free',
+      imagesOnly: true
+    }
+    window.uploadcare.openDialog(null, opts).done((file) => {
+      file.done((info) => {
+        self.setState({src: info.cdnUrl}, self.update)
```

Comment:
`self`? Isn't the `this` from the parent `uploadcare` method still be visible here?